server/core/config/valid/global_cfg.go

```diff
@@ -212,7 +227,8 @@ func NewGlobalCfgFromArgs(args GlobalCfgArgs) GlobalCfg {
 				IDRegex:                   regexp.MustCompile(".*"),
 				BranchRegex:               regexp.MustCompile(".*"),
 				RepoConfigFile:            args.RepoConfigFile,
-				ApplyRequirements:         applyReqs,
+				ApplyRequirements:         commandReqs,
+				ImportRequirements:        commandReqs,
```

Comment:
Should these both be the same?

Reply:
yes. `commandReqs` are generated by user configuration like `--require-mergeable`. it should affect both apply and import requirements.